yt_dlp/extractor/cbc.py

```diff
@@ -180,6 +180,24 @@ class CBCPlayerIE(InfoExtractor):
             'thumbnail': 'http://thumbnails.cbc.ca/maven_legacy/thumbnails/sonali-karnick-220.jpg',
             'chapters': [],
             'duration': 494.811,
+            'categories': [
+                'AudioMobile/All in a Weekend Montreal',
+            ],
+            'tags': [
+                'CBC',
+                'CBC Radio',
+                'Montreal',
+                'Quebec',
+                'All In A Weekend',
+                'Sonali Karnick',
+                'hackathon',
+                'listenmontreal',
+            ],
```

Comment:
```suggestion
            'categories': ['AudioMobile/All in a Weekend Montreal'],
            'tags': 'count:8',
```
Could you please make these changes in the other instances as well?

Reply:
Will be adding the changes to the other tests in a bit. Are tests not supposed to be that exact?

Reply:
In general, tags/categories tests should be formatted to stay in one line. If there are too many to list them in one line, I prefer to use count instead.

Reply:
Ah crap, now I've changed a number of tests to `count:1`. The tests have run through successfully, I'll change it back and re-run the tests tomorrow, been a long day today.
But thanks for the explanation, now I understand.

Reply:
OK, I've corrected it back.